tests/integration/targets/openssl_pkcs12/tasks/main.yml

```diff
@@ -4,6 +4,81 @@
 # and should not be used as examples of how to write Ansible roles #
 ####################################################################
 
-- name: Run tests
-  include_tasks: impl.yml
-  when: pyopenssl_version.stdout is version('17.1.0', '>=')
+- block:
+  - name: Generate private keys
+    openssl_privatekey:
+      path: '{{ output_dir }}/ansible_pkey{{ item }}.pem'
+      size: '{{ default_rsa_key_size_certifiates }}'
+    loop:
+      - 1
+      - 2
+      - 3
+
+  - name: Generate privatekey with password
+    openssl_privatekey:
+      path: '{{ output_dir }}/privatekeypw.pem'
+      passphrase: hunter2
+      cipher: auto
+      size: '{{ default_rsa_key_size }}'
+
+  - name: Generate CSRs
+    openssl_csr:
+      path: '{{ output_dir }}/ansible{{ item }}.csr'
+      privatekey_path: '{{ output_dir }}/ansible_pkey{{ item }}.pem'
+      commonName: www{{ item }}.ansible.com
+    loop:
+      - 1
+      - 2
+      - 3
```

Comment:
```suggestion
    loop: "{{ range(1, 4) }}"
```

Reply:
Implemented in 044294fd3fbed164a23e842fb2a160c8a1a6431e.